src/lru.rs

```diff
@@ -322,11 +327,8 @@ fn _expired(timestamp: Timestamp, now: Timestamp) -> bool {
 pub fn compute_weight<K: HasWeight, V: HasWeight>(key: &K, value: &V) -> Weight {
     // this isn't perfect because it ignores some hashtable and btreeset
     // overhead, but it's a pretty good guess at the memory usage of an entry
-    (3*key.weight()
-     + value.weight()
-     + mem::size_of::<Weight>()
-     + mem::size_of::<Timestamp>()
-     + mem::size_of::<Option<Timestamp>>())
+    (3 * key.weight() + value.weight() + mem::size_of::<Weight>() + mem::size_of::<Timestamp>() +
+     mem::size_of::<Option<Timestamp>>())
```

Comment:
So the official style is "4 spaces, not tabs, 100 col max, but try to keep it to 80 cols".

`rustfmt` also prints weird 

    Rustfmt failed at /home/steve/src/rustcached/src/parser.rs:423: line exceeded maximum length (sorry)

lines, but they're not errors, more like warnings.

Can't wait till it's out of beta, heh.